cupsfilters/pstops.c

```diff
@@ -477,29 +586,31 @@ add_page(pstops_doc_t *doc,		/* I - Document information */
 
 static int				/* O - 1 if selected, 0 otherwise */
 check_range(pstops_doc_t *doc,		/* I - Document information */
```

Comment:
Does not need "doc" in the call scheme any more, only page, Ranges, and pageset.

Reply:
Thanks for pointing this out.I have done the necessary change.